src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -48,7 +48,7 @@ public ArgumentMultimap tokenize() throws ParseException {
     /**
      * Finds all zero-based prefix positions in the given arguments string.
      *
-     * @return           List of zero-based prefix positions in the given arguments string
+     * @return List of zero-based prefix positions in the given arguments string
      */
     private List<PrefixPosition> findAllPrefixPositions() throws ParseException {
         return Arrays.stream(prefixes)
```

Comment:
I just noticed that the `ParseException` is never thrown by`findAllPrefixPositions()` method. I think we can remove the `throw` keyword from the method signature. By doing so, we also need to remove the `throw` keyword from the `tokenize()` method since the `tokenize` is just passing the exception from the `findAllPrefixPositions` (which is never thrown). Btw, this is based on the current master, so please correct me if the current PR actually uses the `ParseException` 😅 

In addition, if we decide to remove the `throw ParseException`, then we should also remove the `throw ParseException` from the junit tests in `ArgumentTokenizerTest` class.

Reply:
this is not by me.